dask_expr/expr.py

```diff
@@ -491,9 +491,9 @@ def _name(self):
     @property
     def columns(self):
```

Comment:
```suggestion
    def columns(self) -> list:
```
Maybe we start being explicit about the expected type here? (Certainly not a blocker)

Reply:
Sounds good to me